Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.Tests/Telemetry/ClientTelemetryTests.cs

```diff
@@ -245,14 +244,116 @@ public async Task CheckIfPayloadIsDividedCorrectlyAsync(int expectedOperationInf
+                                                            ClientTelemetryOptions.RequestLatencyMax,
+                                                            ClientTelemetryOptions.RequestLatencyPrecision);
+                latency.RecordValue(10);
+
+                LongConcurrentHistogram requestcharge = new LongConcurrentHistogram(ClientTelemetryOptions.RequestChargeMin,
+                                                            ClientTelemetryOptions.RequestChargeMax,
+                                                            ClientTelemetryOptions.RequestChargePrecision);
+                requestcharge.RecordValue(11);
+
+                operationInfoSnapshot.TryAdd(opeInfo, (latency, requestcharge));
+            }
+
+            ConcurrentDictionary<CacheRefreshInfo, LongConcurrentHistogram> cacheRefreshInfoSnapshot
+                = new ConcurrentDictionary<CacheRefreshInfo, LongConcurrentHistogram>();
+            for (int i = 0; i < 10; i++)
+            {
+                CacheRefreshInfo crInfo = new CacheRefreshInfo(Regions.WestUS,
+                                                        10,
+                                                        Documents.ConsistencyLevel.Session.ToString(),
+                                                        "databaseName" + i,
+                                                        "containerName",
+                                                        Documents.OperationType.Read,
+                                                        Documents.ResourceType.Document,
+                                                        200,
+                                                        1002,
+                                                        "dummycache");
+
+                LongConcurrentHistogram latency = new LongConcurrentHistogram(ClientTelemetryOptions.RequestLatencyMin,
+                                                            ClientTelemetryOptions.RequestLatencyMax,
+                                                            ClientTelemetryOptions.RequestLatencyPrecision);
+                latency.RecordValue(10);
+
+                cacheRefreshInfoSnapshot.TryAdd(crInfo, latency);
+            }
+
+            try
+            {
+                Task processorTask = Task.Run(() => processor.ProcessAndSendAsync(
+                                                     clientTelemetryProperties,
+                                                     operationInfoSnapshot,
+                                                     cacheRefreshInfoSnapshot,
+                                                     default,
+                                                     new CancellationTokenSource(TimeSpan.FromSeconds(1)).Token));
+
+                await ClientTelemetry.RunProcessorTaskAsync(DateTime.Now.ToString(), processorTask, TimeSpan.FromTicks(1));
```

Comment:
```suggestion
                await Assert.ThrowsAsync<OperationCanceledException>(() -> ClientTelemetry.RunProcessorTaskAsync(DateTime.Now.ToString(), processorTask, TimeSpan.FromTicks(1)));
```

This removes the try/catch and all the other Asserts